app/about/page.tsx

```diff
@@ -1,17 +1,154 @@
-import { Heading, Text } from "@chakra-ui/react";
+import { Flex, Link, Text, Box, VStack, HStack } from "@chakra-ui/react";
+import Heading from "../components/heading";
+import { Headings } from "../data/data";
+import NextLink from "next/link";
 
 const About = () => {
+  const headingData = Headings.about;
   return (
-    <>
-      <Heading>Who are we</Heading>
-      <Text>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eu justo
-        ipsum. Nullam eleifend consectetur mauris ac porta. Fusce venenatis,
-        libero vitae fringilla volutpat, odio diam maximus nibh, eu vestibulum
-        lacus tellus sed felis. Suspendisse potenti. Nulla iaculis sapien vitae
-        iaculis egestas.
+    <Flex
```

Comment:
nit: some of these wrappers (the Flex, Text ones etc) seem like they follow a pattern and could be reusable. something to consider in future work!

Reply:
Thank you and I absolutely agree! I will make everything more reusable-friendly once we finalize the content of this page :)